### PR TITLE
CDK-996: Fixing an issue where the SchemaTool async table creation was n...

### PR DESCRIPTION
**Pull request by prazanna**
...ot waiting for the table to be created before adding new column families

### CODE REVIEW

**Comment by prazanna** [comment]
I will look into why this fails for HADOOP-1 profile

File: kite-data/kite-data-hbase/src/test/java/org/kitesdk/data/hbase/tool/SchemaToolTest.java [hunk 1]
```diff
@@ -83,6 +83,7 @@ public void after() throws Exception {
 
   @Test
   public void testMigrateDirectory() throws Exception {
+    //HBaseTestUtils.util.deleteTable(Bytes.toBytes(tableName));
```
Comment (rdblue) [inline kite-data/kite-data-hbase/src/test/java/org/kitesdk/data/hbase/tool/SchemaToolTest.java]: Accidental?

**Comment by rdblue** [comment]
This looks good to me. I think we should remove the non-functional change to SchemaToolTest though.

**Comment by prazanna** [comment]
Sorry. Removed that.

**Comment by rdblue** [comment]
Merged in https://github.com/kite-sdk/kite/commit/a85dd62ef567fd7f20741522accb94e2d59fbd42. Thanks @prazanna!

**Pull request closed without merging**